searx/engines/base.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+"""
+ BASE (Scholar publications)
+
+ @website     https://base-search.net
+ @provide-api yes with authorization (https://api.base-search.net/)
+
+ @using-api   yes
+ @results     XML
+ @stable      ?
+ @parse       url, title, publishedDate, content
+ More info on api: http://base-search.net/about/download/base_interface.pdf
+"""
+
+from lxml import etree
+from urllib import urlencode
+from searx.utils import searx_useragent
+from cgi import escape
+from datetime import datetime
+import re
+
+categories = ['science']
+
+base_url = 'https://api.base-search.net/cgi-bin/BaseHttpSearchInterface.fcgi'\
+           + '?func=PerformSearch&{query}&boost=oa&hits={hits}&offset={offset}'
+
+# engine dependent config
+paging = True
+number_of_results = 10
+
+
+def request(query, params):
+    # shortcuts for advanced search
+    shorcut_dict = {
```

Comment:
Could you write new shortcuts for the advanced search? These words could cause problems. For example if I want to searcch for the term `cat title` the query would be transformet to `cat dctitle`. That is not what I wanted to search for.

I checked the shortcuts at BASE. It seems to me that it has completelydifferent advanced search syntax. For instance if you want to search for something that is titled French, you would enter `tit:French`. To not to confuse experienced BASE users, I suppose the shortcut keys should be the same as at BASE.